ap/plotting/plotter.py

```diff
@@ -14,45 +14,46 @@
 mplhep = maybe_import("mplhep")
 
 
-def draw_error_stairs(ax: plt.Axes, h: hist.Hist, kwargs: Optional[dict] = None) -> None:
-    if kwargs is None:
-        kwargs = {}
-    norm = kwargs.pop("norm", 1)
-    values = h.values() / norm
-    error = np.sqrt(h.variances()) / norm
+def draw_error_stairs(ax: plt.Axes, h: hist.Hist, norm: float = 1.0, **kwargs) -> None:
+    # compute relative errors
+    rel_error = h.variances()**0.5 / h.values()
+    rel_error[np.isnan(rel_error)] = 0.0
```

Comment:
This function is not only used for the relative errors in the lower panel but also for the Total syst. uncert. in the upper panel. Does that also work?

Reply:
Ah I see, should in principle work since `height` and `bottom` are multiplied by the `baseline`. However, when running the task myself, I don't see any error bands. Do you have the same issue?

Reply:
ok, turns out the hatch is just not drawn for some reason. The computation of the errors looks correct.

Reply:
I checked and the statistical error is just to small (it's mostly in the 0.5 - 1.2% range)